.circleci/config.yml

```diff
@@ -55,9 +55,9 @@ jobs:
       - run: curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
       - run: make setup
       - setup_remote_docker
-      - run: make build_linux
-      - run: make build_master_image
-      - run: make push_master_image
+      - run: make build-linux
```

Comment:
Is there are reason for the change from `_` to `-` ?

Reply:
Usually, the commands are `make value-value` so just to keep the same standard.